app/src/lib/design-system-v2/helpers/ThemeProvider.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { ThemeProvider as StyledComponentsThemeProvider, createGlobalStyle } from "styled-components";
+import { ColorTokens, generateColorTokens } from "../typography/colors";
```

Comment:
We should keep `colors` as a separate folder at same level of typography. Typography should only contain the text related configs (size, weight, family, .....).

Reply:
Done